src/kOS/Screen/TermWindow.cs

```diff
@@ -14,7 +15,8 @@ public class TermWindow : KOSManagedWindow
     {
         private const int CHARSIZE = 8;
         private const string CONTROL_LOCKOUT = "kOSTerminal";
-        private const int CHARS_PER_ROW = 16;
+        private const int FONTIMAGE_CHARS_PER_ROW = 16;
```

Comment:
Renamed to make it clear that the "per row" it's talking about is about how the character pictures are packed into the font image file, not about rows of the terminal.